lib/legacy/class.html

```diff
@@ -33,9 +33,10 @@
      * to ensure that any legacy behaviors can rely on legacy Polymer API on
      * the underlying element.
      *
-     * @param {!(Object|Array)} behaviors Behavior object or array of behaviors.
-     * @param {!HTMLElement|function(new:HTMLElement)} klass Element class.
-     * @return {function(new:HTMLElement)} Returns a new Element class extended by the
+     * @template T
+     * @param {!(PolymerInit|Array<!PolymerInit>)} behaviors Behavior object or array of behaviors.
+     * @param {function(new:T)} klass Element class.
+     * @return {function(new:T)} Returns a new Element class extended by the
```

Comment:
can i get some review here? two things changed:

* we no longer accept an instance of an element, it didn't seem to make any sense unless im missing something (further down it ends up extending what you passed)
* i added a `T` so we accept and return the same type

Unfortunately, closure compiler isn't able to have something like... `T extends HTMLElement`. so this is still kinda wrong but better typed than before i suppose?

cc @rictic @aomarks

Reply:
FYI, Tuesday's when most people will start coming back to work.

Reply:
haha no worries. who knows why im on here on NYE :tada: think its time to celebrate instead

Reply:
Definitely :)

Thanks for your recent contributions, looking forward to more in 2018 😄